abjad/tools/commandlinetools/CheckClassSections.py

```diff
@@ -0,0 +1,169 @@
+#! /usr/bin/env python
+'''
+Validate ### ... ### header order and section contents in classes.
+
+Could be extended to check for other similar errors as needed.
+'''
+
+import os
+import sys
+from abjad.tools.commandlinetools.CommandlineScript import CommandlineScript
+
+
+class CheckClassSections(CommandlineScript):
+    r'''Checks the order and contents of class sections in a path.
+
+    .. shell::
+
+        ajv check-class-sections --help
+
+    '''
+
+    ### CLASS VARIABLES ###
+
+    alias = 'check-class-sections'
+    short_description = (
+        'Check the order and contents of class sections in a path.'
+        )
+    long_description = (
```

Comment:
Drop `Finds` to a new line at the same indentation as `as well as...`. If you wrap the entire string like `long_description = stringtools.normalize(...` here, you'll get cleaner indentation in the `--help` output. Try it for yourself with `ajv check-class-sections --help` to see the inconsistent indentation:

```
17:09 $ ajv check-class-sections --help
usage: check-class-section [-h] [--version] [path]

Check the order and contents of class sections in a path.

positional arguments:
  path        directory tree to be recursed over

optional arguments:
  -h, --help  show this help message and exit
  --version   show program's version number and exit

Finds and lists errors in class section order,
    as well as cases where methods and properties appear
    under incorrect section headers

    If no `path` is given, the current directory is searched.

    Checks that where they appear in classes, the following
    comment headers appear in the given order:

        ### CLASS VARIABLES ###
        ### CONSTRUCTOR ###
        ### INITIALIZER ###
        ### SPECIAL METHODS ###
        ### PRIVATE METHODS ###
        ### PUBLIC METHODS ###
        ### PRIVATE PROPERTIES ###
        ### PUBLIC PROPERTIES ###

    Additionally, this finds cases where methods appear
    under PROPERTIES sections, and vice-versa.
```